lib/explorer/backend/data_frame.ex

```diff
@@ -248,10 +248,18 @@ defmodule Explorer.Backend.DataFrame do
 
         data = container_doc(open, values, close, inspect_opts, &Explorer.Shared.to_string/2)
 
+        type =
+          case Series.dtype(series) do
+            {:datetime, :milli_seconds} -> "datetime[ms]"
+            {:datetime, :micro_seconds} -> "datetime[μs]"
+            {:datetime, :nano_seconds} -> "datetime[ns]"
```

Comment:
Elixir standard library uses  :second | :millisecond | :microsecond | :nanosecond. So maybe we should align?

```suggestion
            {:datetime, :milli_seconds} -> "datetime[ms]"
            {:datetime, :micro_seconds} -> "datetime[μs]"
            {:datetime, :nano_seconds} -> "datetime[ns]"
```